hack/kokoro/upload.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+# Copyright 2021 The Skaffold Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+
+make -f Makefile -f hack/kokoro/Makefile upload-kokoro-release
```

Comment:
Isn't there a danger of using make that the file versions may be earlier than VERSION and so we don't upload the files?  Couldn't we just copy in the `gsutil cp` commands into here?